ssl/quic/quic_channel_local.h

```diff
@@ -451,6 +451,9 @@ struct quic_channel_st {
     /* Has qlog been requested? */
     unsigned int                    use_qlog                            : 1;
 
+    /* Has qlog been requested? */
+    unsigned int is_tserver_ch:1;
```

Comment:
Can this be formatted like the lines above?

Reply:
the line above violates c-style enforced by `util/check-format.pl`. So I've made the change this way to keep c-style check green in CI. The `util/check-format.pl` complains with error:
```
ssl/quic/quic_channel_local.h:452:extra space:    unsigned int                    use_qlog                            : 1;
ssl/quic/quic_channel_local.h:455:extra space:    unsigned int                    is_tserver_ch:1;
```
I will make my change consistent with existing members, and add style-waived label.